### PR TITLE
feat(base-cluster): make speedtest image configurable

### PR DESCRIPTION
**Pull request by tasches**

### CODE REVIEW

File: charts/base-cluster/values.yaml [hunk 1]
```diff
@@ -75,6 +75,7 @@ traefik:
 speedtest:
   enabled: true
   host: speedtest
+  repository: adolfintel/speedtest
```
Comment (mstroecker) [inline charts/base-cluster/values.yaml:78]: Technically it is not a `repository`? At other places in this file, we use the following scheme:

Example from above:

```yaml
image:
      registry: docker.io
      repository: alpine/helm
      tag: 3.12.1
```

**Review by mstroecker — CHANGES_REQUESTED** [review]

File: charts/base-cluster/values.yaml (hunk 1 above)
Comment (tasches) [reply charts/base-cluster/values.yaml:78]: I followed this schema. If you have a look at speedtest.yaml from line 29 to 31 you can see the same structure.

Comment (tasches) [reply charts/base-cluster/values.yaml:78]: helm templated

**Review by mstroecker — APPROVED** [review]
LGTM

**Pull request merged**